Databases/CommandSchedulerDb/Scheduler/Tables/ScheduledCommand.sql

```diff
@@ -15,8 +15,8 @@
 
 
 GO
-CREATE CLUSTERED INDEX [IX_AppliedTime_Clock_Id_FinalAttemptTime_DueTime]
-    ON [Scheduler].[ScheduledCommand]([AppliedTime] ASC, [Clock_Id] ASC, [FinalAttemptTime] ASC, [DueTime] ASC);
+CREATE CLUSTERED INDEX [IX_AggregateId_DueTime_Clock_Id_AppliedTime_FinalAttemptTime]
+    ON [Scheduler].[ScheduledCommand]([AggregateId] ASC, [DueTime] ASC, [Clock_Id] ASC, [AppliedTime] ASC, [FinalAttemptTime] ASC);

```

Comment:
For a new install there must be a entity way of doing it.  

For already installed db, how it gets created varies 

Sent from my iPhone

> On Apr 22, 2016, at 6:35 PM, Jon Sequeira <notifications@github.com> wrote:
> 
> In Databases/CommandSchedulerDb/Scheduler/Tables/ScheduledCommand.sql:
> 
> > @@ -15,8 +15,8 @@
> >  
> >  
> >  GO
> > -CREATE CLUSTERED INDEX [IX_AppliedTime_Clock_Id_FinalAttemptTime_DueTime]
> > -    ON [Scheduler].[ScheduledCommand]([AppliedTime] ASC, [Clock_Id] ASC, [FinalAttemptTime] ASC, [DueTime] ASC);
> > +CREATE CLUSTERED INDEX [IX_AggregateId_DueTime_Clock_Id_AppliedTime_FinalAttemptTime]
> > +    ON [Scheduler].[ScheduledCommand]([AggregateId] ASC, [DueTime] ASC, [Clock_Id] ASC, [AppliedTime] ASC, [FinalAttemptTime] ASC);
> There should be a migration that applies this index as well, no?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>